tests/Microsoft.Identity.Test.iOS.UIAutomation/App.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json"
+                          publicKeyToken="30ad4fe6b2a6aeed"
+                          culture="neutral" />
+        <!-- 
+           Assembly versions can be redirected in application, 
+           publisher policy, or machine configuration files.
+         -->
+
+           <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0"/>
```

Comment:
It seems that you are using Json 9.0.1 already, why do you need a binding redirect?